src/python_be.cc

```diff
@@ -362,9 +363,19 @@ ModelInstanceState::SaveRequestsToSharedMemory(
     const char* id;
     RETURN_IF_ERROR(TRITONBACKEND_RequestId(request, &id));
 
-    uint64_t correlation_id;
-    RETURN_IF_ERROR(
-        TRITONBACKEND_RequestCorrelationId(request, &correlation_id));
+    uint64_t correlation_id_uint = 0;
+    CorrelationId correlation_id;
+
+    auto error =
+        TRITONBACKEND_RequestCorrelationId(request, &correlation_id_uint);
+    if (error != nullptr) {
+      const char* correlation_id_string = "";
```

Comment:
@krishung5 @pskiran1 is this a memory leak not deleting the TRITONSERVER_Error ?

Reply:
Ah.. this is a really great catch, thanks Ryan. I think there will be a memory leak when the error is not deleted. @pskiran1 Can you create a follow-up PR to fix this?

Reply:
Thank you @rmccorm4, I have created a follow-up PR #345 to delete the error.